c++/src/kj/string.h

```diff
@@ -600,40 +725,72 @@ inline ArrayPtr<const char> StringPtr::slice(size_t start, size_t end) const {
   return content.slice(start, end);
 }
 
+inline LiteralStringConst::operator ConstString() const {
+  return ConstString(cStr(), size(), NullArrayDisposer::instance);
```

Comment:
This is nitpicky but `cStr()` is really meant to mean "Give me a null-terminated C string, possibly truncating the string if it contains NUL characters." What you want here is "Give me a pointer to the underlying buffer." The correct method for that is `begin()`.

Admittedly, these methods have the same underlying implementation.